hypothesis-python/src/hypothesis/extra/cli.py

```diff
@@ -84,27 +84,60 @@ def obj_name(s: str) -> object:
             return importlib.import_module(s)
         except ImportError:
             pass
+        classname = None
         if "." not in s:
             modulename, module, funcname = "builtins", builtins, s
         else:
             modulename, funcname = s.rsplit(".", 1)
             try:
                 module = importlib.import_module(modulename)
             except ImportError as err:
+                try:
+                    modulename, classname = modulename.rsplit(".", 1)
+                    module = importlib.import_module(modulename)
+                except ImportError as err:
+                    raise click.UsageError(
+                        f"Failed to import the {modulename} module for introspection.  "
+                        "Check spelling and your Python import path, or use the Python API?"
+                    ) from err
```

Comment:
I think we want to raise this from the *first* `err`, so we can remove the `as err` on line 98.

Reply:
sure